CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
+* kms: added output-only field `primary` to `google_kms_crypto_key` ([#6782](https://github.com/hashicorp/terraform-provider-google-beta/pull/6782))
+* metastore: added `consumers.custom_routes_enabled` to google_dataproc_metastore_service ([#6767](https://github.com/hashicorp/terraform-provider-google-beta/pull/6767))
+* sql: added support for IAM GROUP authentication in the `type` field of `google_sql_user` ([#6787](https://github.com/hashicorp/terraform-provider-google-beta/pull/6787))
+* storagetransfer: made `name` field to be optional in resource `google_storage_transfer_job` so it can be provided by the users ([#6777](https://github.com/hashicorp/terraform-provider-google-beta/pull/6777))
+* vertexai: added `force_destroy` field to `google_vertex_ai_feature_online_store` resource ([#6789](https://github.com/hashicorp/terraform-provider-google-beta/pull/6789))
+* vertexai: added `optimized`, `dedicated_serving_endpoint` and `embedding_management` fields to `google_vertex_ai_feature_online_store` resource ([#6789](https://github.com/hashicorp/terraform-provider-google-beta/pull/6789))
+
+BUG FIXES:
+* container: added check that node_version and min_master_version are the same on create, when running terraform plan ([#6763](https://github.com/hashicorp/terraform-provider-google-beta/pull/6763))
+* container: fixed a bug where disable PDCSI addon `gce_persistent_disk_csi_driver_config ` during creation will result in permadiff in `google_container_cluster` resource ([#6751](https://github.com/hashicorp/terraform-provider-google-beta/pull/6751))
+* container: fixed an issue in which migrating from the deprecated Binauthz enablement bool to the new evaluation mode enum inadvertently caused two cluster update events, instead of none. ([#6785](https://github.com/hashicorp/terraform-provider-google-beta/pull/6785))
+* containerattached: fixed crash when updating a cluster to remove `admin_users` or `admin_groups` in `google_container_attached_cluster` ([#6786](https://github.com/hashicorp/terraform-provider-google-beta/pull/6786))
+* dialogflowcx: fixed a permadiff in the `git_integration_settings` field of `google_diagflow_cx_agent` ([#6756](https://github.com/hashicorp/terraform-provider-google-beta/pull/6756))
+* gkehub2: added field `version` under `configmanagement` instead of a child field `oci` in `google_gke_hub_feature` resource (revert) ([#6755](https://github.com/hashicorp/terraform-provider-google-beta/pull/6755))
+* gkehub: added field `version` under `configmanagement` instead of a child field `oci` in `google_gke_hub_feature` resource ([#6749](https://github.com/hashicorp/terraform-provider-google-beta/pull/6749))
+* monitoring: fixed the index out of range crash in `dashboard_json` perma-diffs for the resource `google_monitoring_dashboard` ([#6750](https://github.com/hashicorp/terraform-provider-google-beta/pull/6750))
```

Comment:
same change here as for https://github.com/hashicorp/terraform-provider-google/pull/16903/files#r1443246122